app-conf/FetcherConf.xml

```diff
@@ -32,6 +32,8 @@
   <fetcher>
     <applicationtype>mapreduce</applicationtype>
     <classname>com.linkedin.drelephant.mapreduce.MapReduceFetcherHadoop2</classname>
+    <!-- If encounter performance issues, try to use fetcher below -->
```

Comment:
Also potentially we should say something about the old fetcher being deprecated and slated to be removed. @akshayrai can comment on this.